examples/next-prisma-starter/src/pages/index.tsx

```diff
@@ -15,56 +27,77 @@ const IndexPage: NextPageWithLayout = () => {
 
   // prefetch all posts for instant navigation
   // useEffect(() => {
-  //   for (const { id } of postsQuery.data ?? []) {
-  //     utils.post.byId.prefetch({ id })
+  //   const allPosts = postsQuery.data?.pages.flatMap((page) => page.items) ?? [];
+  //   for (const { id } of allPosts) {
+  //     void utils.post.byId.prefetch({ id });
   //   }
   // }, [postsQuery.data, utils]);
 
   return (
     <>
       <h1>Welcome to your tRPC starter!</h1>
       <p>
-        Check <a href="https://trpc.io/docs">the docs</a> whenever you get
+        Check <a href="https://trpc.io/?v10">the docs</a> whenever you get
         stuck, or ping <a href="https://twitter.com/alexdotjs">@alexdotjs</a> on
         Twitter.
       </p>
 
       <h2>
-        Posts
+        Latest Posts
         {postsQuery.status === 'loading' && '(loading)'}
       </h2>
-      {postsQuery.data?.map((item) => (
-        <article key={item.id}>
-          <h3>{item.title}</h3>
-          <Link href={`/post/${item.id}`}>
-            <a>View more</a>
-          </Link>
-        </article>
+
+      <button
+        onClick={() => postsQuery.fetchNextPage()}
+        disabled={!postsQuery.hasNextPage || postsQuery.isFetchingNextPage}
+      >
+        {postsQuery.isFetchingNextPage
+          ? 'Loading more...'
+          : postsQuery.hasNextPage
+          ? 'Load More'
+          : 'Nothing more to load'}
+      </button>
+      {postsQuery.data?.pages.map((page, index) => (
+        <Fragment key={index}>
+          {page.items.map((item) => (
+            <article key={item.id}>
+              <h3>{item.title}</h3>
+              <Link href={`/post/${item.id}`}>
+                <a>View more</a>
+              </Link>
+            </article>
+          ))}
+        </Fragment>
```

Comment:
```suggestion
      {postsQuery.data?.pages.map((page, index) => (
        <Fragment key={index}>
          {page.items.map((item) => (
            <article key={item.id}>
              <h3>{item.title}</h3>
              <Link href={`/post/${item.id}`}>
                <a>View more</a>
              </Link>
            </article>
          ))}
        </Fragment>
        <button
        onClick={() => postsQuery.fetchNextPage()}
        disabled={!postsQuery.hasNextPage || postsQuery.isFetchingNextPage}
        >
        {postsQuery.isFetchingNextPage
          ? 'Loading more...'
          : postsQuery.hasNextPage
          ? 'Load More'
          : 'Nothing more to load'}
        </button>
```

It makes for a much better UX for the user to load more when at the bottom of the page, rather than having to scroll all the way back up to load more posts.

Reply:
I wanted to have the latest post at the bottom. 🤷 

This example is not about showcasing great UX, it's only about DX :)